django_project/healthyrivers_base/static/js/views/landing-page.js

```diff
@@ -8,26 +8,36 @@ $(document).ready(function () {
         }
     });
     $.ajax({
-        url: '/api/fish-collections/',
+        url: '/api/fish-summary/',
         dataType: 'json',
         success: function (data) {
             $('#chart-fish').parent().empty().append('<canvas id="chart-fish" width="250px" height="250px"></canvas>');
-            var total_sites = Object.keys(data).length;
-            $('#fish-total-records').html(total_sites);
             var native = 0;
             var non_native = 0;
             var translocated = 0;
-            $.each(data, function (index, key) {
-                if(key['category'] === 'native' || key['category'] === 'indigenous'){
-                    native += 1;
-                }
-                if(key['category'] === 'non-native' || key['category'] === 'alien'){
-                    non_native += 1;
-                }
-                if(key['category'] === 'translocated'){
-                    translocated += 1;
+            $.when($.each(data, function (index, value) {
```

Comment:
hi @ann26 
why don't make return in json?
data['total_fish']
data['category']['native']

etc
so we don't need this loop, just call directly into the data based on the key

example
```
native = data['category']['native'] + data['category']['indigenous']
```